docs/index.rst

```diff
@@ -1,5 +1,5 @@
-Welcome to astro-provider-ray documentation!
-===================================================
+Welcome to Ray provider documentation!
```

Comment:
```suggestion
Airflow Ray Provider
```
Wdyt?

Reply:
I like the idea but lets do it in a separate PR. We will also work with marketing to see if we can get a logo for the provider